constants.ts

```diff
@@ -705,15 +705,16 @@ export const EventSignatures = {
   },
   LSP6: {
     /**
-     * event Executed(
+     * event VerifiedCall(
+     *     address indexed signer,
+     *     uint256 indexed value,
      *     bytes4 indexed selector,
-     *     uint256 indexed value
      * );
      *
-     * signature = keccak256('Executed(bytes4,uint256)')
+     * signature = keccak256('VerifiedCall(address,uint256,bytes4)')
      */
     Executed:
```

Comment:
We should rename the variable here in `constants.ts`, since the LSP6 event name has changed from `Executed` --> `CallVerified`

```suggestion
    VerifiedCall:
```